File: OutliersAPP.API/Models/ApplyForJob.cs
using System;
using System.Collections;
using System.Collections.Generic;
using Microsoft.AspNetCore.Identity;

namespace OutliersAPP.API.Models
{
    public class ApplyForJob 
    {
        public int JobId { get; set; }
        public int UserId { get; set; }
        public Job Job { get; set; }
        public User User { get; set; }
         public string Msg { get; set; }
        public DateTime ApplyDate { get; set; }
        public string JobTitle { get; set; }
        public string UserName { get; set; }
        
    
    }
}

File: OutliersAPP.SPA/src/app/_models/applyForJob.ts
import { Job } from "./job"
import { User } from "./user"

export interface ApplyForJob {
  id: number;
  jobId: number;
  userId: number;
  job: Job;
  user: User;
  msg: string;
  applyDate: Date;
  jobTitle:string;
  userName:string;
}
